.github/workflows/e2e-scheduled.yml

```
@@ -0,0 +1,253 @@
+      contents: write
+      actions: read
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@34e114876b0b11c390a56381ad16ebd13914f8d5 # v4
+        with:
+          # This job only needs .github/scripts/ and .github/actions/failure-notification
+          # from the current tree — no history. The e2e-metrics push below does its git
+          # work in an isolated throwaway clone (see "Persist run stats"), so this
+          # checkout never pushes and doesn't need full history or persisted credentials.
+          persist-credentials: false
+          persist-credentials: false
```

actionlint (1.7.12)

[error] key "persist-credentials" is duplicated in "with" section. previously defined at line:126,col:11. note that this key is case insensitive

(syntax-check)